<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="description" content="Tanevski Lab - Computational biomedical discovery. Machine learning / Artificial intelligence methods for analysis of spatial omics data in Heidelberg"/>
  <meta name="keywords" content="spatial, single-cell, omics, machine learning, ML, optimization , artificial intelligence, AI, heidelberg, germany, proteomics, transcriptomics, metabolomics, translational, computational, biology. biomedicine, medicine" />
  <title>Tanevski Lab | Computational biomedical discovery | Spatial omics | Heidelberg</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="author" content="Jovan Tanevski">

  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">
  <link rel="stylesheet" href="css/custom.css">

  <link rel="icon" type="image/png" href="images/favicon.png">
</head>

<body>
  
<header class="container sticky">
  <div class="row">
    <div class="seven columns">
      <a href="index.html" style="text-decoration: none; color: #000000;">
        <h5>computational<font color="#99001A">biomedical</font><font color="#003399">discovery</font></h5>
      </a>
    </div>
    <div class="five columns" style="word-spacing: 0.75rem; text-align: right;">
      <a href="#main">Research</a>
      <a href="#people">People</a>
      <a href="#tools">Tools</a>
      <a href="#publications">Publications</a>
      <a href="#positions">Positions</a>
    </div>
  </div>
</header>

<div class="container" style="padding-bottom: 2rem;"> 
  <div class="row" style="text-align: center;">
    <h3>Tanevski Lab</h3>
      <img src="images/logo.png" style="width: 15%;">
      <p id="contact" style="font-size: 85%; line-height: 1.3rem;">
      Institute for Computational Biomedicine
      <br/>
      Heidelberg University Hospital
      <br/>
      Im Neuenheimer Feld 130.3, 69120 Heidelberg, Germany
      <br/>
      <a>contact&lt;at&gt;tanevskilab.org</a>
      </p>
  </div>
</div>


<div id="main" class="container">
  <h5>Research</h5>
  <div class="row boxin">
      <p>
        <b>Our group focuses on problem driven development of AI/ML approaches to data exploration, hypothesis generation and computational scientific discovery to facilitate translational biomedicine.</b>
      </p>
      <p >
        We work on representation learning and (un-)supervised analysis of spatial omics data. We develop new and extend existing explainable, scalable and readily deployable methods for multi-view learning, graph neural networks, metaheuristic optimization and optimal transport to: 
      </p>
      <ul>
          <li>
            Identify clinically relevant spatial regions and interactions by explanatory modeling and optimization of global and local tissue/condition specific persistent multicellular patterns.
          </li>
          <li>
            Learn higher order spatial structural and functional organization to form taxonomical models of tissues for comparative analyses and generation of in-silico samples.
          </li>
          <li>
            Integrate spatial multiomics data with databases of prior knowledge to discover context specific mechanistic insighs spanning multiple omics layers.
          </li>
      </ul>
      <p>
        Our interest is to address questions of structure-function relationships in disease, progression and response to treatment. 
      </p>
      <p>
        We value collaborations with clinical, experimental biology groups and groups working on the development of novel methods for the acquisition of spatial omics data. We welcome synergistic collaborations with computational groups towards the construction of more robust theoretical and computational frameworks for the analysis of all aspects of biomedical data and beyond.
      </p>
  </div>
</div>

<div id="people" class="container">
  <h5>People</h5>
  <div class = "row boxin">
      <p><b>Group Leader</b></p>
      <div style="text-align: center;">
        <img class="round" src="images/jovan.jpg" style="max-width: 150px; object-position: 50% 10%;">
        <p>
          <a>
            Jovan Tanevski
          </a>
        </p>
    </div>
  </div>
  <div class = "row boxin">
    <p><b>Members</b></p>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
        <img class="round" src="images/chang.jpg">
        <p>
          <a>
            Chang Lu
          </a>
          <br/>Postdoc
        </p>
      </div>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
      <img class="round" src="images/sebastian.jpg">
      <p>
        <a>
          Sebastian Gonzalez Tirado
        </a>
        <br/>PhD student
      </p>
      
    </div>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
      <img class="round" src="images/aroj.jpg">
      <p>
        <a>
          Aroj Hada 
        </a>
        <br/>PhD student
      </p>
    </div>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
      <img class="round" src="images/leoni.jpeg" style="object-position: 50% 10%;">
      <p>
        <a>
          Leoni Zimmermann 
        </a>
        <br/>Master student
      </p>
    </div>
  </div>
   <div class = "row boxin">
    <p><b>Associated Members</b></p>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
     <img class="round" src="images/philipp.jpg">
        <p>
          <a>
            Philipp Schäfer
          </a>
          <br/>PhD student
        </p>
    </div>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
      <img class="round" src="images/robin.jpg">
        <p>
          <a>
            Robin Fallegger
          </a>
          <br/>PhD student
        </p>
    </div>
    <div class = "two columns" style="max-width: 150px; text-align: center;">
      <img class="round" src="images/chiara.jpg">
        <p>
          <a>
            Chiara Schiller
          </a>
          <br/>PhD student
        </p>
    </div>
  </div>
</div>


<div id = "tools" class="container">
  <h5>Tools</h5>
  <div class="row boxin">
      

      <div class="row">
        <div class="two columns">
          <img class="full" src="images/misty_badge.png">
          <p>
            <b>MISTy</b>
            <br/>
            &#128279; <a href="https://saezlab.github.io/mistyR/" target="_blank">R package and page</a>
            <br/>
            &#128196; <a href="https://doi.org/10.1186/s13059-022-02663-5" target="_blank">Tanevski et al. 2022 Genome Biology</a>
          </p>
        </div>

        <div class="four columns">
          <p>
            The Multiview Intercellular SpaTial modeling framework (MISTy) is an explainable machine learning framework for knowledge extraction and analysis of single-cell, highly multiplexed, spatially resolved data. MISTy facilitates an in-depth understanding of intra- and intercellular relationships. MISTy is a flexible framework able to process a custom number of views describing a different spatial or functional context such as intracellular or broader tissue structure, cell-type composition, functional footprints or anatomical regions.
          </p>
        </div>

        <div class="two columns">
          <img class="full" src="images/kasumi_badge.png">
          <p>
            <b>Kasumi</b>
            <br/>
            &#128279; <a href="https://github.com/jtanevski/kasumi" target="_blank">R package</a>
            <br/>
            &#128196; <a href="https://doi.org/10.1038/s41467-025-59448-0" target="_blank">Tanevski et al. 2025 Nature Communications</a>
          </p>
        </div>

        <div class="four columns">
          <p>
            Kasumi is a method for the identification of spatially localized neighborhoods of intra- and intercellular relationships, persistent across samples and conditions. Kasumi learns compressed explainable representations of spatial omics samples while preserving relevant biological signals that are readily deployable for data exploration and hypothesis generation, facilitating translational tasks.
          </p>
        </div>
      </div>

      <hr>

      <div class="row">
        <div class="two columns">
          <img class="full" src="images/DOT_icon.png">
          <p>
            <b>DOT</b>
            <br/>
            &#128279; <a href="https://saezlab.github.io/DOT/" target="_blank">R package and page</a>
            <br/>
            &#128196; <a href="https://doi.org/10.1038/s41467-024-48868-z" target="_blank">Rahimi et al. 2024 Nature Communications</a>
          </p>
        </div>

        <div class="four columns">
          <p>
            DOT is a method for transferring cell features from a reference single-cell RNA-seq data to spots/cells in spatial omics. It operates by optimizing a combination of multiple objectives using a Frank-Wolfe algorithm to produce a high quality transfer. Apart from transferring cell types/states to spatial omics, DOT can be used for transferring other relevant categorical or continuous features from one set of omics to another, such as estimating the expression of missinng genes or transferring transcription factor/pathway activities.
          </p>
        </div>
      

        <div class="two columns">
          <img class="full" src="images/TOAST_icon.png">
          <p>
            <br/>
            <b>TOAST</b>
            <br/>
            &#128279; <a href="https://github.com/cecca46/TOAST" target="_blank">Python package and page</a>
            <br/>
            &#128196; <a href="https://doi.org/10.1101/2025.04.15.648894" target="_blank">Ceccarelli et al. 2025 bioRxiv</a>
          </p>
        </div>

        <div class="four columns">
          <p>
            TOAST (Topography-aware Optimal Alignment of Spatially-resolved Tissues), is a spatially aware Fused Gromov-Wasserstein (FGW) framework for intra-, intersample and temporal alignment of spatial omics data, which explicitly incorporates spatial constraints into the optimal transport objective. TOAST tranfers annotations accross aligned samples, recovers differentiation trajectories and maps single cells to spatial locations.
          </p>
        </div>

    </div>
  </div>
</div>

<div id="publications" class="container">
  <div class="row boxin">
    <h5>Publications</h5>
    Tanevski, J., Vuillard, L., Ibarra-Arellano, M. A., Schapiro, D., Hartmann, F. J., Saez-Rodriguez, J. <a href="hhttps://doi.org/10.1038/s41467-025-59448-0" target="_blank">Learning tissue representation by identification of persistent local patterns in spatial omics data</a>. <em>Nature Communications</em>:16, 4071 (2025).
    <br/>
    Ceccarelli, F., Lio, P., Saez-Rodriguez, J., Holden, S. B., Tanevski, J. <a href="https://doi.org/10.1101/2025.04.15.648894">Topography Aware Optimal Transport for Alignment of Spatial Omics Data</a>. <em>bioRxiv</em>:2025.04.15.648894 (2025).
    <br/> 
    Schiller, C., Ibarra-Arellano, M. A., Bestak, K., Tanevski, J., Schapiro, D. <a href="https://doi.org/10.1101/2025.03.31.646289">Comparison and Optimization of Cellular Neighbor Preference Methods for Quantitative Tissue Analysis</a>. <em>bioRxiv</em>:2025.03.31.646289 (2025).
    <br/>
    Rahimi, A., Vale-Silva, L.A., Faelth Savitski, M., Tanevski, J., Saez-Rodriguez, J. <a href="https://doi.org/10.1038/s41467-024-48868-z" target="_blank">DOT: a flexible multi-objective optimization framework for transferring features across single-cell and spatial omics</a>. <em>Nature Communications</em> 15, 4994 (2024).
    <br/>
    Dimitrov, D., Schäfer, P.S.L., Farr, E., et al. <a href="https://doi.org/10.1038/s41556-024-01469-w" target="_blank">LIANA+ provides an all-in-one framework for cell–cell communication inference</a>. <em>Nature Cell Biology</em> 26, 1613–1622 (2024).
    <br/>
    Laury, A. R., Zheng, S., Aho, N. et al. <a href="https://doi.org/10.1016/j.modpat.2024.100508" target ="_blank">Opening the black box: spatial transcriptomics and the relevance of AI-detected prognostic regions in high grade serous carcinoma</a>. <em>Modern Pathology</em> 100508 (2024).
    <br/>
    Paton, V., Gabor, A., Ramirez Flores, R.O. et al. <a href="https://doi.org/10.1093/nar/gkae552" target="_blank">Assessing the impact of transcriptomics data analysis pipelines on downstream functional enrichment results</a>. <em>Nucleic Acids Research</em> 52(14) 8100-8111 (2024).
    <br/>
    Wünnemann, F., Sicklinger, F., Bestak, K., et al. <a href="https://doi.org/10.1101/2024.05.20.594955">Spatial omics of acute myocardial infarction reveals a novel mode of immune cell infiltration</a>. <em>bioRxiv</em>:2024.05.20.594955 (2024).
    <br/>
    Heumos, L., Schaar, A.C., Lance, C. et al. <a href="https://doi.org/10.1038/s41576-023-00586-w" target="_blank">Best practices for single-cell analysis across modalities</a>. <em>Nature Reviews Genetics</em> 24, 550–572 (2023).
    <br/>
    Tanevski, J., Ramirez Flores, R.O., Gabor, A. et al. <a href="https://doi.org/10.1186/s13059-022-02663-5" target="_blank">Explainable multiview framework for dissecting spatial relationships from highly multiplexed data</a>. <em>Genome Biology</em> 23, 97 (2022).
    <br/>
    Kuppe, C., Ramirez Flores, R.O., Li, Z. et al. <a href="https://doi.org/10.1038/s41586-022-05060-x" target="_blank">Spatial multi-omic map of human myocardial infarction</a>. <em>Nature</em> 608, 766–777 (2022).
    <br/>
    Gabor, A., Tognetti, M., Driessen, A., Tanevski, J. et al. <a href="https://doi.org/10.15252/msb.202110402" target="_blank">Cell‐to‐cell and type‐to‐type heterogeneity of signaling networks: insights from the crowd</a>. <em>Molecular Systems Biology</em>, 17(10), e10402 (2021).
    <br/>
    Schwabenland, M., Salié, H., Tanevski, J. et al. <a href="https://doi.org/10.1016/j.immuni.2021.06.002" target="_blank">Deep spatial profiling of human COVID-19 brains reveals neuroinflammation with distinct microanatomical microglia-T-cell interactions</a>. <em>Immunity</em> 54(70), 1594-1610.e11 (2021)
    <br/>
    Holland, C.H., Tanevski, J., Perales-Patón, J. et al. <a href="https://doi.org/10.1186/s13059-020-1949-z" target="_blank">Robustness and applicability of transcription factor and pathway analysis tools on single-cell RNA-seq data</a>. <em>Genome Biology</em> 21, 36 (2020).
    <br/>
    Tanevski, J., Nguyen, T., Truong, B. et al. <a href="https://doi.org/10.26508/lsa.202000867" target="_blank">Gene selection for optimal prediction of cell position in tissues from single-cell transcriptomics data</a>. <em>Life Science Alliance</em> 3 (11), e202000867 (2020).
    <br/>
    Tanevski, J., Todorovski, L., Džeroski, S. <a href="https://doi.org/10.1016/j.engappai.2019.103423" target="_blank">Combinatorial search for selecting the structure of models of dynamical systems with equation discovery</a>. <em>Engineering Applications of Artificial Intelligence</em> 89, 103423 (2020).
    <br/>
    Tanevski, J., Todorovski, L., Džeroski, S. <a href="https://doi.org/10.1038/srep34107" target="_blank">Process-based design of dynamical biological systems</a>. <em>Scientific Reports</em> 6, 34107 (2016).
    <br/>
    Tanevski, J., Todorovski, L., Džeroski, S. <a href="https://doi.org/10.1186/s12918-016-0273-4" target="_blank">Learning stochastic process-based models of dynamical systems from knowledge and data</a>. <em>BMC Systems Biology</em> 10, 30 (2016).
  </div>
</div>

<div id="positions" class="container">
  <div class="row boxin">
    <h5>Positions</h5>


    <h6>Interns/Master theses</h6>
    <p>
      We continiously offer opportunities for internships and supervision of master theses. We recommend that the duration of the internships is no shorter than three months.
    </p>


    <h6>PhD</h6>
    <!--<p>
      We are looking for a highly motivated and talented PhD student to join the newly created Translational Spatial Profiling Center at the Heidelberg University Hospital, managed by the Institute for Computational Biomedicine and the Institute for Pathology.
    </p>
    <p>
      In a dynamic and multidisciplinary environment, the candidate will work on the development of novel methods for analysis of state-of-the-art spatially resolved data. The candidate will explore the landscape of explainable machine-learning and optimization approaches and create novel approaches that can be applied in a clinical setting. The work will be motivated by problems arising from a range of translational applications, initially focusing on cancer research, and will be directly supported by data generated within the center.
    </p>
    The candidate should:
    <ul>
      <li>Have either a degree in computer science, engineering sciences, physics or mathematics, or a degree in biological sciences or medicine with previous experience in computational work.</li>
      <li>Be motivated to work on challenging interdisciplinary project with biomedical relevance.</li>
      <li>Have a strong interest and/or knowledge about histo(patho)logy and cancer biology.</li>
      <li>Have experience programming in R or Python.</li>
      <li>Have good English communication skills.</li>
    </ul> -->
  
    We currently have an open PhD position in modeling and integration of multilayered omics data with microscopy/pathology imaging data with focus on patients with heart failure with preserved ejection fraction. For more information see the post on <a href="https://jobrxiv.org/job/heidelberg-university-and-heidelberg-university-hospital-27778-phd-postdoc-in-spatial-omics-for-cardiology/">jobRxiv</a>.

    <hr> 

    <!-- <h6>Postdoc</h6>
    <p>
      We are looking for a postdoctoral researcher for a 2 year project on the topic of Identifying the clinical relevance of higher order tissue organization in the tumor microenvironment from spatial multiplexed omics data. The position is offered as part of the Health + Life Science Alliance Inter-institutional Postdoc Program</a>. The position will be shared between the Tanevski Lab at the Heidelberg University Hospital and the <a href="https://www.dkfz.de/en/systemimmunologie-und-einzelzell-biologie/index.php">Hartmann lab</a> at DKFZ.
    </p>
    <p>
      In this project, we are leveraging the MIBI (multiplexed ion beam imaging) platform established in the Hartmann lab, combined with the experience of the Tanevski Lab in representation learning, optimization and graph-based approaches. Our aim is to identify the different levels of structural and functional organization in tumors from colorectal carcinoma and breast cancer patients to better understand how cellular behavior impacts therapeutic responses and clinical outcomes. Beyond the immediate cellular neighborhood, we will expand the analysis both toward the broader tumor microenvironment and toward capturing higher-level organizational patterns.
    </p>
    The candidate should:
    <ul>
      <li>Have either a degree in computer science, engineering sciences, physics or mathematics, or a degree in biological sciences or medicine with previous experience in computational work.</li>
      <li>Have experience programming in R or Python.</li>
      <li>Interest in spatial omics and cancer biology.</li>
    </ul>  -->
    
    <hr>

    For all PhD and Postdoc postions we offer:
    <ul>
      <li>Work contract and funding according to TV-L with all corresponding social benefits.</li>
      <li>Stimulating and supporting interdisciplinary research environment with access to international research networks.</li>
      <li>Access to state-of-the-art spatial omics and clinical data.</li>
      <li>Access to high performance computing infrastructure.</li>
      <li>Access to further training opportunities offered by the Heidelberg University and Heidelberg University Hospital.</li>
    </ul>

    <p>
      To apply please submit a <b>letter of motivation tailored to the position</b> (1 page), <b>CV</b> and a <b>list of references with contact details</b> to <a>contact&lt;at&gt;tanevskilab.org</a>.
    </p>

  </div>
</div>

<footer class = "container">
  <div class="row" style="text-align: center;">
    <hr>
    &copy; 2024 Tanevski Lab
  </div>
</footer>
</body>
</html>