<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="utf-8">
  <meta name="description" content="Tanevski Lab - Computational biomedical discovery - People" />
  <title>Tanevski Lab - Computational biomedical discovery</title>
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="author" content="Jovan Tanevski">

  <link rel="stylesheet" href="css/normalize.css">
  <link rel="stylesheet" href="css/skeleton.css">
  <link rel="stylesheet" href="css/custom.css">

  <link rel="icon" type="image/png" href="images/favicon.png">
</head>

<body>
  
<header class="container sticky">
  <div class="row">
    <div class="seven columns">
      <a href="index.html" style="text-decoration: none; color: #000000;">
        <h5>computational<font color="#99001A">biomedical</font><font color="#003399">discovery</font></h5>
      </a>
    </div>
    <div class="five columns" style="word-spacing: 0.75rem; text-align: right;">
      <a href="index.html">Home</a>
    </div>
  </div>
</header>

<div class = "container" style = "padding-bottom: 2rem;"> 
  <div class="row" style="text-align: center;">
    <h3>Tanevski Lab</h3>
      <img src="images/logo.png" style="width: 15%;">
      <p id="contact" style="font-size: 85%; line-height: 1.3rem;">
      Institute for Computational Biomedicine
      <br/>
      Heidelberg University Hospital
      <br/>
      Im Neuenheimer Feld 130.3, 69120 Heidelberg, Germany
      <br/>
      <a>contact&lt;at&gt;tanevskilab.org</a>
      </p>
  </div>
</div>

<div id="jovan" class="container">
  <div class="row boxin">
    <h5>Jovan Tanevski</h5>
    
    <div class="three columns">
      <img class="full" src="images/jovan.jpg">
      <p>
        <b>Group Leader</b>
        <br/>
        &#128235; <a>jovan&lt;at&gt;tanevskilab.org</a>
        <br/>
        &#127380; <a href="https://orcid.org/0000-0001-7177-1003" target="_blank">ORCID: 0000-0001-7177-1003</a>
      </p>

    </div>

    <div class="eight columns">
      
      <p>
        I am the group leader of the Computational Biomedical Discovery group at the Institute for Computational Biomedicine and the head of the computational platform of the Translational Spatial Profiling Center at the Heidelberg University Hospital, Heidelberg, Germany. I am also a research associate at the Department of Knowledge Technologies at the Jožef Stefan Institute, Ljubljana, Slovenia and a visiting researcher at the European Bioinformatics Institute (EMBL-EBI) in Hinxton, UK.
      </p>

      <p>
        My research interests are at the intersection of systems sciences and machine learning. I am interested in the development of methods that combine data and prior knowledge. I have a strong background in machine learning and experience in computational systems biology. I am experienced in modeling and design of dynamical systems and their application to problems coming from the domains of systems and synthetic biology. I am currently interested in developing methods for analysis of highly multiplexed and spatially resolved data. I am keen to understand the different patterns of expression within the tissue and bridge them to the underlying dynamical mechanisms that can explain the emergent functional programs and organization.
      </p>
    </div>

  </div>
</div>

<div id="sebastian" class="container">
  <div class="row boxin">
    <h5>Sebastian Gonzalez Tirado</h5>
    
    <div class="three columns">
      <img class="full" src="images/sebastian.jpg">
      <p>
        <b>PhD Student</b>
        <br/>
        &#128235; <a>sebastian.gonzalez&lt;at&gt;embl.de</a>
        <br/>
        &#127380; <a href="https://orcid.org/0000-0002-3093-2402">ORCID: 0000-0002-3093-2402</a>
      </p>
    </div>

    <div class="eight columns">

      <p>
        I studied physics in the National Autonomous University of Mexico. In 2017, I received a scholarship to study abroad at the Georg-August-University of Göttingen, Germany. It was during this time period when I started to specialise in Biophysics and Complex Systems and had the opportunity to write my thesis at the Max Planck Institute for Nonlinear Dynamics on the topic of self-organisation of the social amoeba Dictyostelium discoideum. In 2019 I started a Master of Science in Nanobiophysics at the Technical University of Dresden and studied the translation between different mathematical modelling frameworks for cell motion. From 2021-2024, I was working as a Bioimage Analyst in a joint position between EMBL and GSK. Here I have been developing reusable workflows to study image-based spatial-omics data. I am currently interested in the biological insights from more downstream analysis of spatial data.
      </p>

      </p>
      <p>
        
    </div>

  </div>
</div>


<div id="chang" class="container">
  <div class="row boxin">
    <h5>Chang Lu</h5>
    
    <div class="three columns">
      <img class="full" src="images/chang.jpg">
      <p>
        <b>Associated Postdoc</b>
        <br/>
        &#128235; <a>chang.lu&lt;at&gt;uni-heidelberg.de</a>
        <br/>
        &#127380; <a href="https://orcid.org/0000-0001-6304-2820">ORCID: 0000-0001-6304-2820</a>
      </p>
    </div>

    <div class="eight columns">
      <p>
       I studied biometrics identification at the Beijing University, a topic that gave rise to my interest in the application of computational methods to biological data. In my master studies at the Machine Learning and Data Analytics Lab of Southwestern University, China I gained knowledge of machine learning and data mining and applied them to the prediction of protein function and noise functional annotations. In 2017, I crossed Eurasia to Maastricht, the Netherlands, to pursue my PhD in machine learning and statistic approaches for multi-omics and multimodal image data to analyze the mechanism of cardiometabolic diseases. Starting in October 2022, I continued my research path as a postdoctoral fellow at the Institute for Computational Biomedicine at Heidelberg University, Germany, studying computational integration of cellular circuits and immune cell repertoires in spatial and temporal resolution.
      </p>
    
      <p>
        I am interested in the design and application of computational strategies for spatial multi-omics data to study disease mechanisms in complex diseases. During my PhD, I analysed high-dimensional omics and imaging data of plaque and liver through a combination of computational strategies including statistical inference, machine learning and image processing, to dissect the pathogenesis of  cardiovascular and  liver diseases at the molecular, spatial and temporal level. Currently, I focus on the cellular circuit and immune cell repertoire integration at spatial and temporal resolution.
      </p>
    </div>

  </div>
</div>


<div id="philipp" class="container">
  <div class="row boxin">
    <h5>Philipp Schaefer</h5>
    
    <div class="three columns">
      <img class="full" src="images/philipp.jpg">
      <p>
        <b>Associated PhD Student</b>
        <br/>
        &#128235; <a>philipp.schaefer&lt;at&gt;uni-heidelberg.de</a>
        <br/>
        &#127380; <a href="https://orcid.org/0009-0008-4403-8592">ORCID: 0009-0008-4403-8592</a>
      </p>
    </div>

    <div class="eight columns">
      <p>
       I completed my Bachelor’s degree in Biochemistry at Heinrich-Heine University in Düsseldorf and Michigan State University in the US. Throughout this period, I actively participated in various research endeavors primarily centered around the analysis and characterization of individual proteins. However, during my Master’s program at Heidelberg University, my focus shifted towards investigating biological processes more holistically using large single-cell and spatial multi-omics data sets. In my Master’s thesis, for example, I leveraged single-cell RNA and ATAC-seq data to infer and compare gene regulatory networks underlying the development of the mammalian cerebellum. Now I work on deciphering myeloid subtype diversity in multiple sclerosis supported by the DFG  Priority Programme “Local and Peripheral Drivers of Microglial Diversity and Function” (SPP 2395). I conduct my PhD in the lab of Prof. Dr. Julio Saez-Rodriguez and am co-supervised by Dr. Jovan Tanevski.
      </p>
    
      <p>
        My research interests lie at the intersection of systems immunology and machine learning. I am intrigued by the immense potential of emerging single-cell and spatial multi-omics technologies, which allow us to explore the establishment of biological function under normal circumstances as well as its disruption in various diseases. In this regard, my specific interest lies in developing computational methodologies that can elucidate the underlying regulatory mechanisms responsible for autoimmune disorders like multiple sclerosis.
      </p>
    </div>

  </div>
</div>

<div id="robin" class="container">
  <div class="row boxin">
    <h5>Robin Fallegger</h5>
    
    <div class="three columns">
      <img class="full" src="images/robin.jpg">
      <p>
        <b>Associated PhD Student</b>
        <br/>
        &#128235; <a>robin.fallegger&lt;at&gt;uni-heidelberg.de</a>
        <br/>
        &#127380; <a href="https://orcid.org/0000-0003-3264-6028">ORCID: 0000-0003-3264-6028</a>
      </p>
    </div>

    <div class="eight columns">
      <p>
       Both my Bachelor and Master degrees were both obtained at the Ecole Polytechnique Fédérale de Lausanne in Switzerland. Over the course of my studies, I was able to learn from various fields ranging from oncology, medical engineering to advanced data analysis and machine learning methods. Over the course of my Master’s studies I focused on the latter in a variety of fields such as brain-computer interfaces, image analysis and more importantly genomics and transcriptomics. The aim of my Master thesis at the Computational Biology & Cancer Genomic group (UNIL) was to predict the efficacy of the off-label use of targeted cancer therapies for individual patients by using open source gene essentiality screens (both CRISPR and RNAi).I conduct my PhD in the lab of Prof. Dr. Julio Saez-Rodriguez and am co-supervised by Dr. Jovan Tanevski.
      </p>
    
      <p>
       I am interested in how experimental data and prior biological evidence can be used to generate and address new hypotheses by using computational techniques. In particular, I find it fascinating how data analysis techniques from various fields (such as image analysis and machine learning) can be applied to biological data in order to gain valuable insight into the mechanistic background of biological processes or pathogenesis of various diseases.
    </div>

  </div>
</div>

<div id="chiara" class="container">
  <div class="row boxin">
    <h5>Chiara Schiller</h5>
    
    <div class="three columns">
      <img class="full" src="images/chiara.jpg">
      <p>
        <b>Associated PhD Student</b>
        <br/>
        &#128235; <a>chiara.schiller&lt;at&gt;uni-heidelberg.de</a>
        <br/>
        &#127380; <a href="https://orcid.org/0009-0000-3771-2202">ORCID: 0009-0000-3771-2202</a>
      </p>
    </div>

    <div class="eight columns">
      <p>
       In 2019, I received my BSc in Molecular Biomedicine in Bonn, Germany, and pursued my master studies in Molecular Biosciences in Heidelberg, Germany. During this time, I focused on computational biology in translational biomedical research. I explored the field of single-cell transcriptomics as well as various deconvolution methods of bulk RNA sequencing data. I interned and worked at the DKFZ Heidelberg and at ETH Zurich. In my master thesis with the Schapiro lab, I assessed different spatial neighborhood analysis methods using in silico generated tissue data. In my PhD, my goal is to decipher the spatial architecture of liver tumors using highly multiplexed imaging. I conduct my PhD in the lab of Dr. Denis Schapiro and am co-supervised by Dr. Jovan Tanevski.
      </p>
    
      <p>
        My research is driven by understanding biological mechanisms in diseases, especially by leveraging spatial biology. I am fascinated by the vast amount of machine learning algorithms that can help us to achieve this goal. Therefore, I enjoy exploring and developing suitable computational methodologies to detect structural and functional units in tissue. Here, I understand the uniqueness of diseases and aim towards linking observations to prior knowledge about biological mechanisms or patient information.
      </p>
    </div>

  </div>
</div>


<div id="anon" class="container">
  <div class="row boxin">
    <h5>Join us</h5>
    
    <div class="three columns">
      <img class="full" src="images/anon.jpg">
      <p>
        <b>PhD/Postdoc</b>
        <br/>
        &#128235; <a>contact&lt;at&gt;tanevskilab.org</a>
        <br/>
        &#127380; <a>ORCID: 0000-0000-0000-0000</a>
      </p>
    </div>

    <div class="eight columns">
      <p>
        <a href="index.html#positions">Recruiting now</a>
      </p>

      <p>


      </p>
    </div>

  </div>
</div>

<footer class = "container">
  <div class="row" style="text-align: center;">
    <hr>
    &copy; 2024 Tanevski Lab
  </div>
</footer>
</body>
</html>